chalice/app.py

```diff
@@ -27,6 +28,8 @@
 
 Config.set_config(BucketConfig.NORMAL)
 
+dss.logging.configure_lambda_logging()
```

Comment:
This should be above the Config.set_config() line.

Reply:
Sure